docs/advanced/fields.rst

```diff
@@ -488,3 +488,27 @@ returns a dictionary, where keys are field names, and values are
 
     print(field_names)  # dict_keys(['my_field'])
     print(my_field_meta)  # {'expensive': True}
+
+Field processors
```

Comment:
Let's move this section above the Caching section; it seems important.

Reply:
Or: what do you think about moving it above Items section as well?

Reply:
It seems there's a lot of docs about items, sometimes documenting edge cases; field processors look like a straightforward feature which people need to be aware of.